### DISCUSSION TITLE
Why not defaulting to `bun` runtime?

### DISCUSSION THREAD

**Discussion by kvnwolf** [oven-sh/bun#5779] [Q&A]
This is more a question for @Jarred-Sumner, why do I need to pass the [`--bun`](https://bun.sh/docs/cli/run#bun) flag to use the Bun runtime on locally installed CLI's instead of being it the default?

Also, I think it would be really helpful to have a `bunfig.toml` option to always use the `--bun` flag, maybe something like:

```toml
[run]
bun = true
```

**Comment by melroy89** [comment]
I'm not Jarred. But it uses the shebang when it's a script from like package.json or bin folder. So if the shebang is `#!/usr/bin/env node`, NodeJS get used.

Also there are still some issues still with `bun run` scripts in general. Maybe in the (near) future the Bun runtime will be the default runtime in all cases.
